pkgs/development/libraries/libgnt/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, lib, fetchurl, meson, ninja, pkg-config, python2
+, gtk-doc, docbook-xsl-nons
+, glib, ncurses, libxml2
+}:
+let
+  # Pidgin will get build-time dependency on gtk-doc anyway (through gstreamer).
+  docs = true;
+in
+stdenv.mkDerivation rec {
```

Comment:
```suggestion
, glib, ncurses, libxml2
# Pidgin will get build-time dependency on gtk-doc anyway (through gstreamer).
, buildDocs
}:

stdenv.mkDerivation rec {
```

Reply:
Re-named `docs` to `buildDocs` using `rnix-lsp` :slightly_smiling_face:  and moved to argument as `buildDocs ? true`.

I also removed comment about `gstreamer` which is not part of dependencies for `libgnt` and `gtk-doc` is not pulled indirectly in absence of `gtk-doc` in build inputs.